pkg/config/validation/validation.go

```diff
@@ -1624,7 +1619,13 @@ var ValidateVirtualService = RegisterValidateFunc("ValidateVirtualService",
 
 		allHostsValid := true
 		for _, virtualHost := range virtualService.Hosts {
-			if err := agent.ValidateWildcardDomain(virtualHost); err != nil {
+			var err error
+			if appliesToGateway {
+				err = agent.ValidateWildcardDomainForVirtualServiceBoundToGateway(isSniHost(virtualService), virtualHost)
```

Comment:
@howardjohn Not sure i understand  why vs's hosts should allow `_.`

Reply:
The original issue has the context: https://github.com/istio/istio/issues/51285. It is to allow the Istio multi-network sni-dnat format to be matched for users that want to use a normal gateway instead of auto-passthrough

Reply:
I agree this is kind of borderline for backport though since its kind of a feature. I am fine closing this. It did merge in 1.22 though (https://github.com/istio/istio/pull/52659) so we maybe need to revert that for consistency...?

Reply:
Making sure this doesn't go stale. Should we revert 1.22 or merge for this for consistency?

Reply:
This got closed for being stale - given this change appears to have been included in 1.22.4 and 1.22.5 I'm guessing we should consider reopening and merging to get consistent with published releases?

Reply:
+1 for consistency